docs/source/staking/stake_management.rst

```diff
@@ -0,0 +1,259 @@
+Stake Management
+-----------------
+
+Several administrative operations can be performed on active stakes:
+
++----------------------+-------------------------------------------------------------------------------+
+| Action               |  Description                                                                  |
++======================+===============================================================================+
+|  ``restake``         | Manage automatic reward re-staking                                            |
++----------------------+-------------------------------------------------------------------------------+
+|  ``prolong``         | Prolong an existing stake's duration                                          |
++----------------------+-------------------------------------------------------------------------------+
+|  ``winddown``        | Manage winding down of stakes                                                 |
++----------------------+-------------------------------------------------------------------------------+
+|  ``divide``          | Create a new stake from part of an existing one                               |
++----------------------+-------------------------------------------------------------------------------+
+|  ``increase``        | Increase an existing stake's value                                            |
++----------------------+-------------------------------------------------------------------------------+
```

Comment:
Here is missing some commands that mention below (like merge, remove unused). I guess it was missed long time ago but if possible to copy it here will be good

Reply:
@KPrasch I added these, it seems like my changes have been removed...

Reply:
```
commit a9ad104a5589b109fa2905581bd84fce24909bd3 (HEAD -> reprise)
Author: derekpierre <derek.pierre@gmail.com>
Date:   Thu Feb 4 12:11:53 2021 -0500

    Add some context information about NU and its uses.

commit 0dca73b34245f290befab265b0e143d42aa2e73c
Author: derekpierre <derek.pierre@gmail.com>
Date:   Thu Feb 4 11:31:20 2021 -0500

    Clarify FAQ to explain that participation in WorkLock is closed. Also remove CoinList FAQ entry.

commit f8362590e8d4cb0c524c8a2939cc17678310e902
Author: derekpierre <derek.pierre@gmail.com>
Date:   Wed Feb 3 16:04:05 2021 -0500

    Collapse Python API and Contracts API into one APIS section in sidebar.

commit 751fac4bceb618f6ad43d457854635104f14f801
Author: derekpierre <derek.pierre@gmail.com>
Date:   Tue Feb 2 16:41:44 2021 -0500

    Use bullet points for each characteristic of the network to make them more pronounced.

commit 0208f432d5478fd0dee92a11561c51b264d8ec61
Author: derekpierre <derek.pierre@gmail.com>
Date:   Tue Feb 2 16:27:11 2021 -0500

    Mostly reordering of specific page content; addition of helpful/contextual links.
```